tests/test_graphql_api/test_reservation/test_create.py

```diff
@@ -16,14 +26,20 @@
 
 
 @pytest.mark.parametrize("arm", ["suomi_fi", "heltunnistussuomifi"])
+@freezegun.freeze_time("2021-01-01")
 def test_create_reservation__prefilled_with_profile_data(graphql, settings, arm: str):
     # given:
     # - Prefill setting is on
     # - There is a reservation unit in the system
```

Comment:
Should include in this text that "the reservation unit has reservable time spans".